sharedTest/build.gradle

```diff
@@ -31,6 +31,7 @@ android {
 dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
     implementation 'com.google.android.material:material:1.11.0'
+    implementation "androidx.datastore:datastore-preferences:1.0.0"
```

Comment:
We shouldn't have to add this dependency here if your library already uses it

Reply:
Oww it was a save for laptop repair, I guess pushing to origin is not very healthy.

Reply:
These changes will be amended please ignore last commit.